requirements/common.in

```diff
@@ -1,27 +1,23 @@
 # Packages that are shared between deployment and dev environments.
 gunicorn==20.0.4
-whitenoise[brotli]==5.0.1
-
-# Used by Whitenoise to provide Brotli-compressed versions of static files.
-Django==3.0.7
+whitenoise[brotli]  # Used by Whitenoise to provide Brotli-compressed versions of static files.
+Django==3.0.8
```

Comment:
@camd  not everything is unpinned; I suspect unpinning Django is a larger effort.